src/agentscope/models/tongyi_model.py

```diff
@@ -13,9 +14,13 @@
 
 from ..utils.monitor import MonitorFactory
 from ..utils.monitor import get_full_name
-from ..utils import QuotaExceededError
 from ..constants import _DEFAULT_API_BUDGET
 
+# The models in this list require that the roles of messages must alternate
+# between "user" and "assistant".
+# TODO: add more models
+SPECIAL_MODEL_LIST = ["qwen-turbo", "qwen-plus", "qwen1.5-72b-chat"]
```

Comment:
qwen-max seems to have the same problem.
https://help.aliyun.com/zh/dashscope/developer-reference/api-details?spm=a2c4g.11186623.0.i22

Reply:
yes, only qwen-max-1201 needs that the first and the last role should be users